test/Microsoft.ML.Tests/TrainerEstimators/CalibratorEstimators.cs

```diff
@@ -137,5 +137,101 @@ private sealed class CalibratorTestData
             }
         }
 
+        /// <summary>
+        /// Test to confirm calibrator estimators work with classes
+        /// where order of label and score columns are reversed, and
+        /// where 
+        /// </summary>
+        [Fact]
+        public void TestNonStandardCalibratorEstimatorClasses()
+        {
+            var mlContext = new MLContext(0);
+            // Store different possible variations of calibrator data classes.
+            IDataView[] dataArray = new IDataView[]
+            {
+                mlContext.Data.LoadFromEnumerable<CalibratorTestInputReversedOrder>(
+                    new CalibratorTestInputReversedOrder[]
+                    {
+                        new CalibratorTestInputReversedOrder { Score = 10, Label = true },
+                        new CalibratorTestInputReversedOrder { Score = 15, Label = false }
+                    }),
+                mlContext.Data.LoadFromEnumerable<CalibratorTestInputUniqueScoreColumnName>(
+                    new CalibratorTestInputUniqueScoreColumnName[]
+                    {
+                        new CalibratorTestInputUniqueScoreColumnName { Label = true, ScoreX = 10 },
+                        new CalibratorTestInputUniqueScoreColumnName { Label = false, ScoreX = 15 }
+                    }),
+                mlContext.Data.LoadFromEnumerable<CalibratorTestInputReversedOrderAndUniqueScoreColumnName>(
+                    new CalibratorTestInputReversedOrderAndUniqueScoreColumnName[]
+                    {
+                        new CalibratorTestInputReversedOrderAndUniqueScoreColumnName { ScoreX = 10, Label = true },
+                        new CalibratorTestInputReversedOrderAndUniqueScoreColumnName { ScoreX = 15, Label = false }
+                    })
+            };
+
+            // When label and/or score columns are different from their default names ("Label" and "Score", respectively), they
+            // need to be manually defined as done below.
+            // Successful training of estimators and transforming with transformers indicate correct label and score columns
+            // have been found.
+            for (int i = 0; i < dataArray.Length; i++)
```

Comment:
Here, I also attempted to test the variability of the label column between the four calibrator trainers. However, there are inconsistencies between these trainers that prevented me from doing this smoothly. For one, [FixedPlattCalibratorEstimator ](https://github.com/dotnet/machinelearning/blob/ec159ec41c774ef1ae5ecc2a8db3cd4aea3ae55e/src/Microsoft.ML.Data/Prediction/CalibratorCatalog.cs#L363-L374)doesn't have a `labelColumn` parameter, which means only the default "Label" column name can be used with this estimator. In addition, "scoreColumnName" is used in PlattCalibratorEstimator, but in the others "scoreColumn" is used. In a separate PR I will make the necessary changes to fix the problems described.